pkgs/applications/misc/please-cli/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, curl
+, fetchFromGitHub
+, jq
+, makeBinaryWrapper
+, please-cli
```

Comment:
How come a derivation has itself as input ? I guess this line needs to be removed.

Reply:
It's necessary for `testers.testVersion`

Reply:
Fair enough ;) thanks !